### PR TITLE
Reduce nanogpt quantization calib size to speed up test

### PR DESCRIPTION
**Pull request by yizhuoz004**

### CODE REVIEW

File: tripy/examples/nanogpt/quantization.py [hunk 1]
```diff
@@ -43,10 +45,10 @@ def modelopt_quantize(model_hf, quant_mode):
     else:
         raise NotImplementedError(f"Unsupported quantization mode: {quant_mode}")
 
-    calib_size = 512
+    calib_size = 64
     batch_size = 1
     if quant_mode == "int4-weight-only":
-        calib_size = 32
+        calib_size = 16
```
Comment (pranavm-nvidia) [inline tripy/examples/nanogpt/quantization.py:51]: Why do we use a smaller size for INT4?

File: tripy/examples/nanogpt/quantization.py [hunk 2]
```diff
@@ -43,10 +45,10 @@ def modelopt_quantize(model_hf, quant_mode):
     else:
         raise NotImplementedError(f"Unsupported quantization mode: {quant_mode}")
 
-    calib_size = 512
+    calib_size = 64
```
Comment (pranavm-nvidia) [inline tripy/examples/nanogpt/quantization.py:48]: Does this impact the accuracy?

**Review by pranavm-nvidia — APPROVED** [review]

File: tripy/examples/nanogpt/quantization.py (hunk 1 above)
Comment (yizhuoz004) [reply tripy/examples/nanogpt/quantization.py:51]: int4 quantization is very slow, modelopt recommends using a smaller calib size with a bigger batch size.

File: tripy/examples/nanogpt/quantization.py (hunk 2 above)
Comment (yizhuoz004) [reply tripy/examples/nanogpt/quantization.py:48]: No, the outputs are all the same.

**Pull request merged**